examples/plot_simulate_beta.py

```diff
@@ -0,0 +1,132 @@
+"""
+===============================
+09. Simulate beta modulated ERP
+===============================
+
+This example demonstrates how event related potentials (ERP) are modulated
+by prestimulus beta events.
```

Comment:
Can you give a brief description of the experiment and why the model might be appropriate here, or to answer what question? So we can motivate folks with similar data to re-use this model and/or to answer similar questions.